src/components/InvertedFlatList/BaseInvertedFlatList.js

```diff
@@ -25,11 +26,15 @@ const propTypes = {
 
     // Should we measure these items and call getItemLayout?
     shouldMeasureItems: PropTypes.bool,
+
+    // Should we remove the clipped sub views?
+    removeClippedSubviews: PropTypes.bool,
```

Comment:
It might make it a little longer but should (ha) we keep the `should` naming convention ? Like `shouldRemoveClippedSubviews`.

Reply:
I think that would be ok.